src/config.ts

```diff
@@ -687,6 +687,10 @@ export const formatters = {
   "isort": {
     "command": "isort",
     "args": ["--quiet", "-"]
-  }
+  },
 
+  "gofmt": {
+    "command": "gofmt",
+    "args": ["gofmt", "-s", "%filepath"]
```

Comment:
There is no need to add `gofmt` in "args". `gofmt` is a command.

Reply:
> There is no need to add `gofmt` in "args". `gofmt` is a command.

Yup, you're right. It's funny because it works nevertheless.
I'll remove the `gofmt` argument.